common/app/navigation/helpers/UrlHelpers.scala

```diff
@@ -64,8 +70,13 @@ object UrlHelpers {
 
   def readerRevenueLinks(implicit request: RequestHeader): List[NavLink] =
     List(
-      NavLink("Make a contribution", getReaderRevenueUrl(SupportContribute, SideMenu)),
-      NavLink("Subscribe", getReaderRevenueUrl(SupportSubscribe, SideMenu), classList = Seq("js-subscribe")),
+      NavLink("Support us", getReaderRevenueUrl(SupportContribute, SideMenu)),
+      NavLink(
+        "Print subscriptions",
+        if (Edition(request).id == "UK") getReaderRevenueUrl(SupporterCTA, SideMenu)
+        else getReaderRevenueUrl(PrintCTAWeekly, SideMenu),
+        classList = Seq("js-print-subscription"),
+      ),
```

Comment:
Does this get used on the old header & the new header?

Reply:
Yes, but on Mobile that’s always what we want.

Reply:
It should also trickle through to AMP. cc @michaelbjacobson